### PR TITLE
Remove multithreading as we strongly encourage xmp

### PR DESCRIPTION
**Pull request by jysohn23**

### CODE REVIEW

**Review by zcain117 — APPROVED** [review]

**Review by JackCaoG — COMMENTED** [review]
Thanks! Could you update `.circleci/test.sh` as well(it is commented out right now)?

File: docker/common.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ function run_deployment_tests() {
   export XRT_WORKERS="localservice:0;grpc://localhost:40934"
   export CC=clang-8 CXX=clang++-8
 
-  time python /pytorch/xla/test/test_train_mnist.py
+  time python /pytorch/xla/test/test_train_mp_mnist.py --fake_data
```
Comment (taylanbil) [inline docker/common.sh:8]: if the previous command has early stopping (using convergence), and this doesn't, it may have different behavior.

Comment (jysohn23) [reply docker/common.sh:8]: No early stopping pre-existing. This is mainly a very simple test to block deployment of wheels if we smell any smoke.

**Comment by jysohn23** [comment]
> Thanks! Could you update `.circleci/test.sh` as well(it is commented out right now)?

Done.

**Review by taylanbil — APPROVED** [review]

**Review by zcain117 — APPROVED** [review]

**Review by JackCaoG — APPROVED** [review]
Thanks @jysohn23 !

**Pull request merged**